sp800-63c/sec5_assertions.md

```diff
@@ -5,40 +5,43 @@
+8. Authentication Time - A timestamp indicating when the IdP last verified the presence of the subscriber at the IdP through a primary authentication event (if available).
 
 Assertions MAY also include the following information:
 
-- Key binding - Public key or key identifier of a key held by the subscriber to demonstrate their binding with the assertion.
-- Attribute values and attribute claims - Information about the subscriber.
-- Attribute metadata - Additional information about one or more subscriber attributes, such as that described in [[NISTIR 8112]](#nistir8112).
+1. Key binding - Public key or key identifier of a key held by the subscriber to demonstrate their binding with the assertion.
+2. Attribute values and attribute claims - Information about the subscriber.
+3. Attribute metadata - Additional information about one or more subscriber attributes, such as that described in [[NISTIR 8112]](#nistir8112).
 
-Assertions SHOULD specify the AAL when an authentication event is being asserted and IAL when identity proofed attributes or claims based thereon are being asserted. The IAL and AAL MAY be specified in an alternate form, such as a composite level of assurance. If not specified, the RP SHALL NOT assign any specific IAL or AAL to the assertion.
+Assertions SHOULD specify the AAL when an authentication event is being asserted and IAL when identity proofed attributes or claims based thereon are being asserted. If not specified, the RP SHALL NOT assign any specific IAL or AAL to the assertion.
 
 Assertions MAY include additional attributes. Refer to [Section 6](#sec6) for privacy requirements on presenting attributes in assertions. The RP MAY fetch additional identity attributes from the IdP in one or more separate transactions using an authorization credential issued alongside the original assertion. The ability to successfully fetch such additional attributes SHALL NOT be treated as equivalent to processing of the assertion.
 
 Although details vary based on the exact federation protocol in use, an assertion SHOULD be used only to represent a single login event at the RP. After the RP consumes the assertion, [session management](sp800-63b.html#sec7) by the RP comes into play; the assertion SHALL NOT be used past the expiration time contained therein. However, the expiration of the session at the RP MAY occur prior to the expiration of the assertion. See [Section 4.3](#federation-session) for more information.
 
 ### 5.1. Assertion Binding
 
-An assertion can be classified based on whether presentation by a claimant of an assertion reference or the assertion itself is sufficient for establishing the binding between the subscriber and the assertion, or if a stronger binding is required.
+ Assertion binding can be classified based on whether:
+ - presentation by a claimant of an assertion reference; or
```

Comment:
Breaking this up into bullets makes this read unevenly. The first bullet doesn't stand alone.